src/Defra.PTS.Web.UI/Views/Shared/_FooterPartial.cshtml

```diff
@@ -70,3 +77,15 @@
         </div>
     </div>
 </footer>
+
+<!-- Language toggle script -->
+<script>
+    function toggleLanguage() {
```

Comment:
Not sure why we've written this, the logic to change language is already there with the cookie? Please explain

Reply:
this is the toggle function for Welsh Language